src/planner/plan/PlanNode.cpp

```diff
@@ -349,6 +349,15 @@ std::ostream& operator<<(std::ostream& os, PlanNode::Kind kind) {
     return os;
 }
 
+SingleInputNode::SingleInputNode(QueryContext* qctx, Kind kind, const PlanNode* dep)
+    : SingleDependencyNode(qctx, kind, dep) {
+    if (dep != nullptr) {
+        readVariable(dep->outputVarPtr());
+    } else {
+        inputVars_.emplace_back(nullptr);
```

Comment:
The `releaseSymbols()` directly to use the element of inputVars_, need to add check in `releaseSymbols()`.

Reply:
Has checked in `releaseSymbols()`